static/js/pages/instances/InstanceList.tsx

```diff
@@ -218,127 +217,133 @@ const InstanceList: FC = () => {
-                      <p>
-                        Looks like there are no instances in this project!
-                        Create a new instance with the &quot;Add instance&quot;
-                        button above.
-                      </p>
-                      <p>
-                        <a
-                          className="p-link--external"
-                          href="https://linuxcontainers.org/lxd/docs/latest/instances/"
-                          target="_blank"
-                          rel="noreferrer"
-                        >
-                          Read the documentation on instances
-                        </a>
-                      </p>
-                    </Col>
-                  </Row>
-                )
-              }
-            />
-          </Col>
-          {detailInstance && (
-            <Col size={4} className="u-hide--medium u-hide--small">
-              <InstanceDetailPanel
-                instance={detailInstance}
-                notify={notify}
-                onClose={() => setSelected()}
+    <main className="l-main">
+      <div className="p-panel">
+        <div className="p-panel__header">
+          <Row className="p-panel__title p-instance-header">
+            <Col size={2}>
+              <h4>
+                {visibleInstances.length}{" "}
+                {visibleInstances.length === 1 ? "Instance" : "Instances"}
+              </h4>
+            </Col>
+            <Col size={8}>
+              <Row>
+                <Col size={3}>
+                  <SearchBox
+                    id="filter-query"
+                    type="text"
+                    onChange={(value) => {
+                      setQuery(value);
+                    }}
+                    placeholder="Search"
+                    value={query}
+                  />
+                </Col>
+                <Col size={2}>
+                  <Select
+                    id="filter-state"
+                    onChange={(v) => {
+                      setStatus(v.target.value);
+                    }}
+                    options={[
+                      {
+                        label: "All statuses",
+                        value: "any",
+                      },
+                      ...instanceStatusOptions,
+                    ]}
+                    value={status}
+                  />
+                </Col>
+                <Col size={3}>
+                  <Select
+                    id="filter-type"
+                    onChange={(v) => {
+                      setType(v.target.value);
+                    }}
+                    options={[
+                      {
+                        label: "Containers and VMs",
+                        value: "any",
+                      },
+                      ...instanceTypeOptions,
+                    ]}
+                    value={type}
+                  />
+                </Col>
```

Comment:
The filters and the button look a little funny with screen width between 1000 and 1200 px.

I think we should give the button a col size of 3 at least. The filter and search can be together in one col with size 7. Then the two filters can have a fixed width and the search bar can adjust its width to fill the col-7. Maybe consult with design, as the responsive behaviour is not specified. But from the screens, I assume the filters are fixed width and the search input adjusts.

Reply:
Ok, let's first hear from @al-bakalova if your assumptions are correct, and then, once the responsive behaviour becomes clearer, I'll adapt the sizes accordingly.